deep_agent/src/guardrails/client.py

```
@@ -122,12 +128,12 @@ async def _call_guardian(
         verdict = _extract_verdict(raw)
         is_safe = not verdict.lower().startswith("yes")
         logger.info("guardian_check", context=context, verdict=verdict, is_safe=is_safe)
+        if guardrails_circuit_state() == "half-open":
+            close_guardrails_circuit()
         return is_safe, verdict
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)